src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/ExternalMovement.java

```diff
@@ -80,11 +104,11 @@ public static class PK implements Serializable {
     @JoinColumn(name = "ARREST_AGENCY_LOC_ID")
     private AgencyLocation arrestAgencyLocation;
 
-    @ManyToOne(optional = false, fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "FROM_AGY_LOC_ID")
```

Comment:
these are indeed nullable, movements can be to and from cities and other entities. Making it mandatory would mean this is an inner join when it should be an outer join